matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/model/EventEntity.kt

```diff
@@ -56,10 +57,10 @@ internal open class EventEntity(@Index var eventId: String = "",
 
     companion object
 
-    fun setDecryptionResult(result: MXEventDecryptionResult) {
+    fun setDecryptionResult(result: MXEventDecryptionResult, clearEvent: JsonDict? = null) {
```

Comment:
This change is strange, let me see the rest of the changes to understand why it is done